benchmark/module/module-loader.js

```diff
@@ -18,8 +18,10 @@ function main(conf) {
   var n = +conf.thousands * 1e3;
   for (var i = 0; i <= n; i++) {
     fs.mkdirSync(benchmarkDirectory + i);
-    fs.writeFileSync(benchmarkDirectory + i + '/package.json', '{"main": "index.js"}');
-    fs.writeFileSync(benchmarkDirectory + i + '/index.js', 'module.exports = "";');
+    fs.writeFileSync(benchmarkDirectory + i + '/package.json',
+      '{"main": "index.js"}');
```

Comment:
This styling makes me cringe a bit too. IMHO I'd say either again style it like the `spawn()` change from `benchmark/compare.js` or at least line up this line with the start of the first argument.